tests/conftest.py

```diff
@@ -28,7 +32,7 @@ def pytest_runtest_makereport(item, call):
 
 
 @pytest.fixture(scope="session")
-def webserver():
+def webserver() -> Any:
     """Fixture that starts a local web server."""
     from .webserver import WebServer
```

Comment:
This should be able to return something other than `Any`, as it is a yield fixture

Reply:
done used the type `Generator[WebServer, None, None]`